ui/src/app/app-routing.module.ts

```diff
@@ -214,6 +215,16 @@ const routes: Routes = [
                   }
                 }
               },
+              {
+                path: 'data/datasets',
+                component: DataSetComponent,
+                data: {
+                  title: 'DataSet Page',
```

Comment:
I’d probably do `Dataset` or `Data Set` rather than `DataSet` for anything user facing? Keri would have a solid answer there?